pkg/asset/internal/templates.go

```diff
@@ -513,4 +523,92 @@ spec:
     port: 2379
     protocol: TCP
 `)
+
+	KubeFlannelCfgTemplate = []byte(`apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kube-flannel-cfg
+  namespace: kube-system
+  labels:
+    tier: node
+    app: flannel
+data:
+  cni-conf.json: |
+    {
+      "name": "cbr0",
+      "type": "flannel",
+      "delegate": {
+        "isDefaultGateway": true
+      }
+    }
+  net-conf.json: |
+    {
+      "Network": "10.2.0.0/16",
+      "Backend": {
+        "Type": "vxlan"
+      }
+    }
+`)
+
+	KubeFlannelTemplate = []byte(`apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: kube-flannel
+  namespace: kube-system
+  labels:
+    tier: node
+    app: flannel
+spec:
+  template:
+    metadata:
+      labels:
+        tier: node
+        app: flannel
+    spec:
+      hostNetwork: true
+      containers:
+      - name: kube-flannel
+        image: quay.io/coreos/flannel-git:v0.6.1-60-g4b752d1-amd64
```

Comment:
Because there has not yet been a release of flannel which contains required changes. /cc @philips @tomdee -- the changes included in this release are passing k8s conformance tests, is there some additional criteria we should meet to get the changes into a release?